tracer/src/Datadog.Tracer.Native/iast/dataflow.cpp

```diff
@@ -79,6 +79,7 @@ static const WSTRING _fixedAssemblyExcludeFilters[] = {
     WStr("testhost.*"),
     WStr("Oracle.ManagedDataAccess"),
     WStr("DelegateDecompiler*"),
+    WStr("FluentValidation*"),
```

Comment:
Why is this needed?

Reply:
Not really needed, but convenient, for performance reasons